filebeat/tests/system/test_modules.py

```diff
@@ -150,7 +150,10 @@ def run_on_file(self, module, fileset, test_file, cfgfile):
                              bufsize=0).wait()
 
         # List of errors to check in filebeat output logs
-        errors = ["Error loading pipeline for fileset"]
+        errors = [
+            "Error loading pipeline for fileset",
```

Comment:
```suggestion
            "error loading pipeline for fileset",
```

Reply:
This will be implemented outside this PR as discussed.